llvm/lib/Target/AMDGPU/AMDGPUSubtarget.cpp

```diff
@@ -81,7 +81,7 @@ AMDGPUSubtarget::getOccupancyWithWorkGroupSizes(uint32_t LDSBytes,
   // workgroups, maximum number of waves, and minimum occupancy. The opposite is
   // generally true for the minimum group size. LDS or barrier ressource
   // limitations can flip those minimums/maximums.
-  const auto [MinWGSize, MaxWGSize] = getFlatWorkGroupSizes(F);
+  const auto &[MinWGSize, MaxWGSize] = FlatWorkGroupSizes;
```

Comment:
```suggestion
  const auto [MinWGSize, MaxWGSize] = FlatWorkGroupSizes;
```
Don't see why this gained a reference